spec/requests/api/v1/accounts_spec.rb

```diff
@@ -147,6 +147,27 @@
       end
     end
 
+    context 'when user tries to follow their own account' do
+      let(:locked) { false }
+      let(:other_account) { user.account }
+
+      it 'returns http forbidden' do
+        expect(response).to have_http_status(403)
+      end
+
+      it 'returns JSON with an error message' do
+        json = body_as_json
```

Comment:
You might want to rebase this -- the `body_as_json` helper is no longer available (use `response.parsed_body` directly).

Separatetly, you can you combine/chain the three assertions together so we can assert multiple things about the same response (avoid multiple round trips, multiple factories, etc).

Reply:
Noted. Will do.